CrowServe/CrowServe.cpp

```diff
@@ -78,8 +78,17 @@ bool Socket::receive() {
             std::cout << "received message header, type: " << message_header.message_type << " size: " << message_header.message_size << "\n";
             // TODO: figure out protocol-specific message handling
             char buffer[1600];
+            auto p = &buffer;
```

Comment:
Probably just use `&buffer` inline

Reply:
lol - I actually switched to using `&buffer` and just forgot to remove this line. Thanks